### PR TITLE
add factory method for creating F[Log] from implicit LogOf instance

### PR DESCRIPTION
**Pull request by dfahritdinov**

### CODE REVIEW

File: core/src/main/scala/com/evolutiongaming/catshelper/LogOf.scala [hunk 1]
```diff
@@ -19,6 +21,9 @@ object LogOf {
 
   def summon[F[_]](implicit F: LogOf[F]): LogOf[F] = F
 
+  def logger[F[_]: LogOf, C: ClassTag]: F[Log[F]] = apply[F].apply(implicitly[ClassTag[C]].runtimeClass)
```
Comment (t3hnar) [inline core/src/main/scala/com/evolutiongaming/catshelper/LogOf.scala:24]: could we cal it `log` after the class `Log` ?

**Comment by t3hnar** [comment]
Nice idea!

File: core/src/main/scala/com/evolutiongaming/catshelper/LogOf.scala (hunk 1 above)
Comment (dfahritdinov) [reply core/src/main/scala/com/evolutiongaming/catshelper/LogOf.scala]: done

File: core/src/main/scala/com/evolutiongaming/catshelper/LogOf.scala [hunk 2]
```diff
@@ -21,9 +21,9 @@ object LogOf {
 
   def summon[F[_]](implicit F: LogOf[F]): LogOf[F] = F
 
-  def logger[F[_]: LogOf, C: ClassTag]: F[Log[F]] = apply[F].apply(implicitly[ClassTag[C]].runtimeClass)
+  def log[F[_]: LogOf, C: ClassTag]: F[Log[F]] = apply[F].apply(implicitly[ClassTag[C]].runtimeClass)
```
Comment (t3hnar) [inline core/src/main/scala/com/evolutiongaming/catshelper/LogOf.scala:24]: regarding this - I propose to add extension method to `Log` class, and reuse that method, so `ClassTag` functionality is available in both cases

Comment (dfahritdinov) [reply core/src/main/scala/com/evolutiongaming/catshelper/LogOf.scala]: done

**Comment by t3hnar** [comment]
Thanks

**Pull request merged**